pkg/local_object_storage/sawmill/forest.go

```diff
@@ -0,0 +1,109 @@
+package sawmill
+
+import (
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	cidSDK "github.com/nspcc-dev/neofs-sdk-go/container/id"
```

Comment:
Two same imports there :smiley:

Reply:
Wow, it is unexpected `Go` doesn't raise a compile error here, actually.

Reply:
Fixed.